examples/platforms/da15000/Makefile.am

```diff
@@ -0,0 +1,111 @@
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
+
+
+lib_LIBRARIES                             = libopenthread-da15000.a
+
+libopenthread_da15000_a_CPPFLAGS          = \
+    -I$(top_srcdir)/include                 \
+    -I$(top_srcdir)/examples/platforms      \
+    -I$(top_srcdir)/src/core                \
+    -I$(top_srcdir)/third_party/dialog/DialogSDK/bsp/include\
+    -I$(top_srcdir)/third_party/dialog/DialogSDK/bsp/peripherals/include\
+    -I$(top_srcdir)/third_party/dialog/DialogSDK/bsp/adapters/include\
+    -I$(top_srcdir)/third_party/dialog/DialogSDK/bsp/memory/include\
+    -I$(top_srcdir)/third_party/dialog/DialogSDK/interfaces/ftdf/include\
+    -I$(top_srcdir)/third_party/dialog/DialogSDK/bsp/config\
+    -I$(top_srcdir)/third_party/dialog/DialogSDK\
+    -I$(top_srcdir)/third_party/dialog/DialogSDK/interfaces/ftdf/src\
+    -include$(top_srcdir)/examples/platforms/da15000/custom_config_qspi.h\
+    -Wno-unknown-pragmas\
+    -Wno-sign-compare\
+    -Wno-unused-function\
+    -Wno-unused-parameter\
+    -Wno-shadow\
+    -Wno-type-limits\
+    -Wno-unused-variable\
+    -Wno-missing-field-initializers\
+    -fno-strict-aliasing\
+    $(NULL)
+
+
+libopenthread_da15000_settings.cpp: $(srcdir)/../utils/settings.cpp
+	cp $? $@
+
+CLEANFILES = libopenthread_da15000_settings.cpp
+
+libopenthread_da15000_a_SOURCES          = \
+    alarm.c                                \
+    flash.c                                \
+    logging.c                              \
+    platform.c                             \
+    radio.c                                \
+    random.c                               \
+    vector.c                               \
+    cstartup.c                             \
+    uart.c                                 \
+    cortex-m_cstartup.c                    \
+    startup-gcc.c                          \
+    libopenthread_da15000_settings.cpp     \
+    clock.c                                \
+    misc.c                                 \
+./../../../third_party/dialog/DialogSDK/bsp/memory/src/qspi_automode.c \
```

Comment:
A few nits:

1. Avoid relative paths.  Use `@top_builddir@` instead.  For example:
```
    @top_builddir@/third_party/dialog/DialogSDK/bsp/memory/src/qspi_automode.c \
```
2. Indent each entry by 4 spaces.
3. Align `\` with lines above and below.